### PR TITLE
Jennylreid patch 2

### PR DESCRIPTION
**Pull request by jennylreid**
Added section on Impact on localization

Still needs work. Want it to fit the existing style of the doc

### CODE REVIEW

File: translation-function/interpolation.md [hunk 1]
```diff
@@ -129,3 +129,35 @@ While there are a lot of options going with the defaults should get you covered.
...
+Use interpolation sparingly to minimize impact on localization. Interpolation must be used for values that can only be known at runtime, such as 
+* Time stamps
+* User-inputted data
+
+Fundamentally what interpolation does is concatenate pieces of text. When translating into other languages concatenation can cause real problems, unless you are using a multilingual grammar engine, which is rare. Sentences split into sentence fragments and programmatcially constructed based soley on English grammar rules are difficult and sometimes impossible to translate. 
+
+Example
+
+Suppose you want to use interpolation to replace the value for {credit card type name} in the following key
+
```
Comment (MickMonaghan) [inline translation-function/interpolation.md:144]: I'd change the variable name to '{paymentMethod}', becuase:
- in your example below, the variable can be replaced with a credit card name, or 'Paypal account'
- also, variable names will never have spaces

File: translation-function/interpolation.md [hunk 2]
```diff
@@ -129,3 +129,35 @@ While there are a lot of options going with the defaults should get you covered.
...
+```javascript
+{
+    "key": "Alle Beträge werden dem {{credit card type name}} für dieses Konto in Rechnung gestellt."
+}
+```
+// -> "Alle Beträge werden dem Kreditkarte für dieses Konto in Rechnung gestellt." <- 'dem Kreditkarte' should be 'der Kreditkarte'
+
+// -> "Alle Beträge werden dem PayPal-Konto für dieses Konto in Rechnung gestellt." <- 'dem PayPal-Konto' is correct
+
+This is just one simple example of a very complex localization problem.
```
Comment (MickMonaghan) [inline translation-function/interpolation.md:163]: I suggest you provide a solution to this issue - as in the developer should instead create two separate strings:

1. "All fees will be charged to the credit card on file for this account."
2. "All fees will be charged to the Paypal account on file for this account."

and at runtime figure out which one to display

File: translation-function/interpolation.md (hunk 1 above)
Comment (jennylreid) [reply translation-function/interpolation.md:146]: done

File: translation-function/interpolation.md (hunk 2 above)
Comment (jennylreid) [reply translation-function/interpolation.md:168]: thanks

**Pull request merged**